src/applications/check-in/hooks/useGetCheckInData.jsx

```diff
@@ -184,6 +185,11 @@ const useGetCheckInData = ({
             return;
           }
 
+          if (preCheckinAlreadyCompleted(payload.appointments)) {
+            setPreCheckinComplete(true);
```

Comment:
it won't let me add a suggestion for some reason, but this should be `setPreCheckinComplete(window, true)`